include/triton/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVMBase.h

```diff
@@ -88,6 +88,8 @@ class ElementwiseOpConversionBase : public ConvertOpToLLVMPattern<SourceOp> {
       // encoding not available
       return resultVals;
     Attribute baseEncoding = encoding;
+    if (isa<AMDMfmaEncodingAttr>(baseEncoding))
```

Comment:
are sliced of Mfma okay? Shouldn't this check go line 95

Reply:
slice of mfma is ok after  https://github.com/triton-lang/triton/pull/3870

Reply:
Can we add some comments on why skipping here to be clear?